pysat/_files.py

```diff
@@ -13,59 +19,50 @@
 class Files(object):
     """Maintains collection of files plus associated methods.
 
-    Interfaces with the `list_files` method for a given instrument
-    support module to create an ordered collection of files in time,
-    used primarily by the pysat.Instrument object to identify files
-    to be loaded. The Files class mediates access to the files by
-    datetime and contains helper methods for determining the presence of
-    new files and filtering out empty files.
-
-    Parameters
-    -----------
-    sat : pysat.Instrument
-        Instrument object
-    manual_org : boolean
-        If True, then pysat will look directly in pysat data directory
-        for data files and will not use default /platform/name/tag/inst_id
-        (default=False)
-    directory_format : string or NoneType
-        directory naming structure in string format. Variables such as
+    Attributes
```

Comment:
Probably worth going through this docstring and separating parameters and attributes, but this approach was easier when resolving merge conflicts.